### ISSUE TITLE
introduce a requirements.txt

### ISSUE THREAD

**Issue by BjoernLudwig** [bangxiangyong/connect-agentMET4FOF-smartupunit#2]
We should introduce a `requirements.txt` and maybe use lines like `-r datareceiver/requirements.txt`.

**Comment by BeSeeTek** [comment]
done

**Comment by BeSeeTek** [comment]
>  maybe use lines like `-r datareceiver/requirements.txt`.

we should defently do that since i added uncertainties to datareceiver just some seconds ago and have to update the pushed req.txt now :(

**Comment by bangxiangyong** [comment]
i have tried adding : 
```
-r datareceiver/requirements.txt.
-r agentMET4FOF/requirements.txt.
```

but pycharm doesn't immediately recognises them to recommend the installation of the packages..

**Comment by bangxiangyong** [comment]
resolved via 2b22574076bd2e4500fc1d53985fdefc3bbe9177
